src/app.js

```diff
@@ -1,21 +1,38 @@
 'use strict';
 
-/**
- * Implement sum function:
- *
- * Function takes 2 numbers and returns their sum
- *
- * sum(1, 2) === 3
- * sum(1, 11) === 12
- *
- * @param {number} a
- * @param {number} b
- *
- * @return {number}
- */
-function sum(a, b) {
-  // write code here
-  return a + b;
-}
+const http = require('http');
+const url = require('url');
+const fs = require('fs');
 
-module.exports = sum;
+const server = http.createServer((req, res) => {
+  const BASE_URL = `http://${req.headers.host}`;
+  const normalizedUrl = new url.URL(req.url, BASE_URL).pathname;
+  let fileName = normalizedUrl.slice(1).replace('file/', '');
```

Comment:
You should validate if the URL actually starts with '/file/'. The current implementation will just replace 'file/' anywhere in the URL.